frontend/src/BudgetModeDisplay.tsx

```diff
@@ -20,23 +25,40 @@ export function BudgetModeDisplay({
     currentFleetSupply,
     maxFleetSupply,
     unitCounts,
+    isFighterUpgraded,
+    shipCapacityUsed,
+    maxShipCapacity,
+    spaceDockFighterBonus,
 }: Props) {
+    const shipCapacityRemaining =
+        maxShipCapacity +
+        spaceDockFighterBonus -
+        shipCapacityUsed -
+        getFighterCount(unitCounts);
+
     return [
         {
             label: "Resources Remaining",
             value: resourceBudget - getTotalCost(unitCounts),
         },
         {
-            label: "Capacity Remaining",
+            label: "Production Capacity Remaining",
             value: capacityBudget - getTotalCapacity(unitCounts),
         },
         {
             label: "Fleet Supply Remaining",
             value: getFleetSupplyRemaining(
                 currentFleetSupply,
                 maxFleetSupply,
-                unitCounts
+                unitCounts,
+                isFighterUpgraded ? Math.max(0 - shipCapacityRemaining, 0) : 0
```

Comment:
Had to puzzle over this, and see it work in action, before I could convince myself it was correct (it is!). Probably just sleep-deprived :coffee: 

But consider: 
`isFighterUpgraded && shipCapacityRemaining < 0 ? shipCapacityRemaining : 0`

Neither is wrong, so pick your fave.

Reply:
Honestly same, and I'm the one that wrote it 🤦‍♀️ yours is definitely easier on the brain! Updated to that with a teeny adjustment to flip the sign